### PR TITLE
Fixup inject

### PR DESCRIPTION
**Pull request by kbrock**
Just minor tweaks to the syntax.

They can probably be all ignored
If there are any in particular you don't want, just say the word

### CODE REVIEW

File: lib/benchmark/ips/job.rb [hunk 1]
```diff
@@ -260,7 +259,7 @@ def run_benchmark
 
           final_time = Timing.now
 
-          measured_us = measurements_us.inject(0) { |a,i| a + i }
+          measured_us = measurements_us.inject(&:+)
```
Comment (evanphx) [inline lib/benchmark/ips/job.rb]: I'm almost positive this change is a lot slower than the old way.

Comment (kbrock) [inline lib/benchmark/ips/job.rb]: I remembered something about that as well. But looks like for all but 1.8.7, the symbol form is 10% faster https://github.com/JuanitoFatas/fast-ruby/pull/105

Here is what I found:

```bash
[master] fast-ruby/code $ chruby 1.8.7
[master] fast-ruby/code $ ruby enumerable/inject-sum-vs-block.rb 
[...]
Comparison:
          inject sum:        0.1 i/s
        inject block:        0.0 i/s - 1.74x slower
[master] fast-ruby/code $ chruby 1.9.3
[master] fast-ruby/code $ ruby enumerable/inject-sum-vs-block.rb 
Comparison:
        inject block:        1.6 i/s
          inject sum:        1.5 i/s - 1.10x slower
[master] fast-ruby/code $ chruby 2.0.0
[master] fast-ruby/code $ ruby enumerable/inject-sum-vs-block.rb 
Comparison:
        inject block:        1.5 i/s
          inject sum:        1.3 i/s - 1.20x slower
[master] fast-ruby/code $ chruby 2.1.5
[master] fast-ruby/code $ ruby enumerable/inject-sum-vs-block.rb 
[...]
Comparison:
        inject block:        1.5 i/s
          inject sum:        1.3 i/s - 1.15x slower
[master] fast-ruby/code $ chruby 2.2.4
[master] fast-ruby/code $ ruby enumerable/inject-sum-vs-block.rb 
[...]
Comparison:
        inject block:        1.6 i/s
          inject sum:        1.4 i/s - 1.12x slower

File: lib/benchmark/timing.rb [hunk 2]
```diff
@@ -35,14 +35,9 @@ def self.stddev(samples, m=nil)
     # @param [Integer] resample_times Resample times, defaults to 100.
     # @return [Array] Resampled samples.
     def self.resample_mean(samples, resample_times=100)
-      resamples = []
-
-      resample_times.times do
-        resample = samples.map { samples[rand(samples.size)] }
-        resamples << Timing.mean(resample)
+      resample_times.times.map do
+        Timing.mean(samples.map { samples[rand(samples.size)] })
```
Comment (zenspider) [inline lib/benchmark/timing.rb]: Please use `{}` here to follow Weirich method

**Comment by kbrock** [comment]
@zenspider changed `map` to use `{}` per Jim Weirich even though it spans multiple lines.

I tried on a single line and even though it wasn't too many characters, it was just too dense for me.

**Comment by zenspider** [comment]
I don't see any reason why the CI failed. It borks on bundle against json only on the 1.8 flavors. @evanphx ? Any clue? 

I also can't figure out how to simply restart the CI and have it give it another go.

**Comment by evanphx** [comment]
Weird. Looks fine otherwise, I'll see about CI.

**Pull request merged**